state/migrations/20231019153023_cleanup_dead_invites_test.go

```diff
@@ -0,0 +1,127 @@
+package migrations
+
+import (
+	"embed"
+	_ "embed"
+	"encoding/json"
+	_ "github.com/lib/pq"
+	"github.com/matrix-org/sliding-sync/state"
+	"github.com/pressly/goose/v3"
+	"reflect"
+	"testing"
+)
+
+//go:embed 20231019153023_cleanup_dead_invites.sql
+var deadInviteMigration embed.FS
+
+func TestDeadInviteCleanup(t *testing.T) {
+	db, close := connectToDB(t)
+	defer close()
+	store := state.NewStorageWithDB(db, false)
+
+	// Alice and Bob have invites to rooms X and Y.
+	// Chris has an invite to room Z.
+	t.Log("Alice is joined to room X; Bob to room Y. Chris isn't joined to any rooms.")
```

Comment:
Log line is confusing and wrong? Wrong position in the test and Bob is banned from room Y, not joined.

Reply:
Oops. Thanks.